modules/publicize/ui.php

```diff
@@ -18,6 +18,10 @@ function __construct() {
 
 		$this->publicize = $publicize = new Publicize;
 
+		add_action( 'init', array( $this, 'init' ) );
+	}
+
+	function init() {
```

Comment:
That's the thing—we can't call `current_user_can` until the `init` hook without throwing the notices.